---
title: Data Repositories
category: Research-Data-Management
layout: default
docs_css: markdown
---
A **data repository** is a location where digital objects are stored and documented, and which enable the separate publication and archiving of these objects. Data access can be either open or restricted to a group of users.

## Benefits of publishing data in a repository
* **Back-up** (i.e. the practice of keeping additional copies of your research data in a separate physical or cloud locations from your files in storage).
* Increased **data discoverability** and **accessibility**, and thus potential **data reuse**.
* Increased **visibility** and **citations** as repositories can be harvested by **search engines** such as:
    * [BASE](https://www.base-search.net/about/en/index.php)
    * [DataCite Search](https://search.datacite.org/)
    * [Google Dataset Search](https://datasetsearch.research.google.com/)
    * [LIVIVO - ZB MED Search Portal for Life Sciences](https://www.livivo.de/)

## Criteria to select a trusted repository
Below are listed criteria you might want to consider when selecting a repository where to publish your data:
* Does the repository provide:
    * **Open Access** to non-sensitive data?
    * The **terms of use** and **licenses** of the data?
    * A **policy** to help researchers determine whether their data needs will be met?
* Does the repository use a Persistent Identifier (PID) system such as assigning a **Digital Object Identifier** (DOI) to submitted datasets?
* Is the repository **certified** (e.g. by [CoreTrustSeal](https://www.coretrustseal.org/) or [nestor](https://www.langzeitarchivierung.de/Webs/nestor/EN/Home/home_node.html)) or does it support a repository **standard**?
* Further criteria: costs, metadata to be provided, repository size, data upload restriction(s), landing pages, guidance on data citation, long-term sustainability.

## Order of preference for selecting a repository
1. A well-established repository in your discipline that offers a specific scope (i.e. supporting a specific metadata schema).
2. Your institution’s repository (if available).
3. One of the repositories recommended by your funding organization.
4. A cost-free interdisciplinary repository (e.g. [Figshare](https://figshare.com/), [Zenodo](https://zenodo.org/)).
5. Another repository that you can search for using the above-mentioned criteria in a repository finder. 

## Well-established repositories in microbiology
Below are liste well-established repositories in microbiology. For each repository, the FAIRsharing and re3data pages are linked. On the FAIRsharing page, you will find information such as which journals endorse the repository (under "Collections & Recommendations" and then "In Policies"). On the re3data page, you will find information such as the above-mentioned criteria to select a trusted repoository.

| Data type 	| Data repository 	| FAIRsharing 	| re3data 	|
|---	|---	|---	|---	|
| **Crystallographic data** 	| Cambridge Structural Database ([CSD](https://www.ccdc.cam.ac.uk/solutions/csd-core/components/csd/)) 	| [FAIRsharing](https://fairsharing.org/FAIRsharing.vs7865) 	| [re3data](https://www.re3data.org/repository/r3d100010197) 	|
| **Geospatial data** 	| Malaria Atlas Project ([MAP](https://malariaatlas.org/)) 	| / 	| [re3data](https://www.re3data.org/repository/r3d100011838) 	|
| **Image data** 	| [BioImage Archive](https://www.ebi.ac.uk/bioimage-archive/) 	| [FAIRsharing](https://fairsharing.org/FAIRsharing.x38D2k) 	| [re3data](https://www.re3data.org/repository/r3d100013949) 	|
|  	| [Cell Image Library](http://www.cellimagelibrary.org/home) 	| [FAIRsharing](https://fairsharing.org/FAIRsharing.8t18te) 	| [re3data](https://www.re3data.org/repository/r3d100000023) 	|
| **Linked genotype and phenotype data** 	| European Genome-phenome Archive ([EGA](https://ega-archive.org/)) 	| [FAIRsharing](https://fairsharing.org/FAIRsharing.mya1ff) 	| [re3data](https://www.re3data.org/repository/r3d100011242) 	|
| **Linked Open Data (LOD)** 	| [The Linked Open Data Cloud](https://lod-cloud.net/) 	| / 	| / 	|
| **Macromolecular structures** 	| Worldwide Protein Data Bank ([wwPDB](http://www.wwpdb.org/)) 	| [FAIRsharing](https://fairsharing.org/FAIRsharing.mckkb4) 	| [re3data](https://www.re3data.org/repository/r3d100011104) 	|
|  	| Biological Magnetic Resonance Data Bank ([BMRB](https://bmrb.io/)) 	| [FAIRsharing](https://fairsharing.org/FAIRsharing.p06nme) 	| [re3data](https://www.re3data.org/repository/r3d100010191) 	|
| **Electron microscopy data** 	| Electron Microscopy Data Bank ([EMDB](https://www.ebi.ac.uk/emdb/)) 	| [FAIRsharing](https://fairsharing.org/FAIRsharing.651n9j) 	| [re3data](https://www.re3data.org/repository/r3d100010562) 	|
|  	| Electron Microscopy Public Image Archive ([EMPIAR](https://www.ebi.ac.uk/empiar/)) 	| [FAIRsharing](https://fairsharing.org/FAIRsharing.dff3ef) 	| [re3data](https://www.re3data.org/repository/r3d100012356) 	|
| **Microbiome data** 	| [MGnify](https://www.ebi.ac.uk/metagenomics/) 	| [FAIRsharing](https://fairsharing.org/FAIRsharing.dxj07r) 	| [re3data](https://www.re3data.org/repository/r3d100011192) 	|
| **Nucleic acid sequences** 	| [GenBank](https://www.ncbi.nlm.nih.gov/genbank/) 	| [FAIRsharing](https://fairsharing.org/FAIRsharing.9kahy4) 	| [re3data](https://www.re3data.org/repository/r3d100010528) 	|
|  	| DNA Data Bank of Japan ([DDBJ](https://www.ddbj.nig.ac.jp/index-e.html)) 	| [FAIRsharing](https://fairsharing.org/FAIRsharing.a308a0) 	| [re3data](https://www.re3data.org/repository/r3d100010218) 	|
|  	| European Nucleotide Archive ([ENA](https://www.ebi.ac.uk/ena/browser/home)) 	| [FAIRsharing](https://fairsharing.org/FAIRsharing.dj8nt8) 	| [re3data](https://www.re3data.org/repository/r3d100010527) 	|
|  	| Sequence Read Archive ([SRA](https://www.ncbi.nlm.nih.gov/sra)) 	| [FAIRsharing](https://fairsharing.org/FAIRsharing.g7t2hv) 	| [re3data](https://www.re3data.org/repository/r3d100010775) 	|
|  	| Genome Sequence Archive ([GSA](https://ngdc.cncb.ac.cn/gsa/)) 	| [FAIRsharing](https://fairsharing.org/FAIRsharing.tdhkc6) 	| [re3data](https://www.re3data.org/repository/r3d100012342) 	|
| **Metagenomes** 	| [MG-RAST](https://www.mg-rast.org/) 	| / 	| [re3data](https://www.re3data.org/repository/r3d100011031) 	|
| **Genetic polymorphism** 	| European Variation Archive ([EVA](https://www.ebi.ac.uk/eva/)) 	| [FAIRsharing](https://fairsharing.org/FAIRsharing.6824pv) 	| [re3data](https://www.re3data.org/repository/r3d100011553) 	|
| **Functional genomics** 	| Gene Expression Omnibus ([GEO](https://www.ncbi.nlm.nih.gov/geo/)) 	| [FAIRsharing](https://fairsharing.org/FAIRsharing.5hc8vt) 	| [re3data](https://www.re3data.org/repository/r3d100010283) 	|
|  	| [ArrayExpress](https://www.ebi.ac.uk/biostudies/arrayexpress) 	| [FAIRsharing](https://fairsharing.org/FAIRsharing.6k0kwd) 	| [re3data](https://www.re3data.org/repository/r3d100010222) 	|
|  	| [GenomeRNAi](http://www.genomernai.org/) 	| / 	| [re3data](https://www.re3data.org/repository/r3d100011089) 	|
|  	| European Genome-phenome Archive ([EGA](https://ega-archive.org/)) 	| [FAIRsharing](https://fairsharing.org/FAIRsharing.mya1ff) 	| [re3data](https://www.re3data.org/repository/r3d100011242) 	|
|  	| Database of Interacting Proteins ([DIP](https://dip.doe-mbi.ucla.edu/dip/Main.cgi)) 	| [FAIRsharing](https://fairsharing.org/FAIRsharing.qje0v8) 	| [re3data](https://www.re3data.org/repository/r3d100010670) 	|
|  	| [IntAct](https://www.ebi.ac.uk/intact/home) 	| [FAIRsharing](https://fairsharing.org/FAIRsharing.d05nwx) 	| [re3data](https://www.re3data.org/repository/r3d100010671) 	|
|  	| Japanese Genotype-phenotype Archive ([JGA](https://www.ddbj.nig.ac.jp/jga/index-e.html)) 	| [FAIRsharing](https://fairsharing.org/FAIRsharing.pwgf4p) 	| [re3data](https://www.re3data.org/repository/r3d100012689) 	|
|  	| [PubChem](https://pubchem.ncbi.nlm.nih.gov/) 	| [FAIRsharing](https://fairsharing.org/FAIRsharing.qt3w7z) 	| [re3data](https://www.re3data.org/repository/r3d100010129) 	|
|  	| Genomic Expression Archive ([GEA](https://www.ddbj.nig.ac.jp/gea/index-e.html)) 	| [FAIRsharing](https://fairsharing.org/FAIRsharing.hESBcy) 	| [re3data](https://www.re3data.org/repository/r3d100013187) 	|
|  	| Genome-Wide Association Studies ([GWAS](https://www.ebi.ac.uk/gwas/)) Catalog 	| [FAIRsharing](https://fairsharing.org/FAIRsharing.blUMRx) 	| / 	|
| **Protein sequences** 	| [UniProt](https://www.uniprot.org/) 	| [FAIRsharing](https://fairsharing.org/FAIRsharing.s1ne3g) 	| [re3data](https://www.re3data.org/repository/r3d100010357) 	|
| **Protein-protein interactions** 	| [STRING](https://string-db.org/) 	| [FAIRsharing](https://fairsharing.org/FAIRsharing.9b7wvk) 	| [re3data](https://www.re3data.org/repository/r3d100010604) 	|
| **Proteomes** 	| PRoteomics IDEntifications ([PRIDE](https://wwwdev.ebi.ac.uk/pride/)) Archive database 	| [FAIRsharing](https://fairsharing.org/FAIRsharing.e1byny) 	| [re3data](https://www.re3data.org/repository/r3d100010137) 	|
| **Quantitative and predictive food microbiology** 	| [ComBase](https://www.combase.cc/index.php/en/) 	| / 	| [re3data](https://www.re3data.org/repository/r3d100010878) 	|
| **Scientific texts** 	| [PUBLISSO – Repository for Life Sciences](https://repository.publisso.de/) 	| [FAIRsharing](https://fairsharing.org/4357) 	| [re3data](https://www.re3data.org/repository/r3d100013523) 	|
| **Species interaction data** 	| Global Biotic Interactions ([GloBI](https://www.globalbioticinteractions.org/)) 	| / 	| / 	|
| **Standardized bacterial information** 	| Bacterial Diversity Metadatabase ([BacDive](https://bacdive.dsmz.de/)) 	| [FAIRsharing](https://fairsharing.org/FAIRsharing.aSszvY) 	| [re3data](https://www.re3data.org/repository/r3d100013060) 	|
| **Vertebrate-virus network** 	| [VIRION](https://viralemergence.github.io/virion/) 	| / 	| / 	|
{: .table .table-hover}

## Data publishing in the PUBLISSO – Repository for Life Sciences (FRL)

#### Repository details
* **Data type**: scientific texts and research data from the fields of medicine, health, nutritional, environmental and agricultural sciences
* **Open Access**: yes
* **Terms of use & License**: yes
* **Persistent Identifier (PID) system**: yes (DOI)
* **Certification or repository standard**: no
* **Policy**: yes ("[Data Policy](https://www.publisso.de/fileadmin/user_upload/Downloads/Data_Policy_Fachrepositorium_Lebenswissenschaften_Okt_2019.pdf)", German only)
* **Archiving**: yes
* **Costs**: free of charge
* **Data curation & quality control**: no
* **Data guarantee**: availability for at least 10 years and transfer to a long-term archive
* **Recommended publication formats for research data**: see on this [page](https://www.publisso.de/en/research-data-management/rd-archiving/)

#### Steps to deposit data in the FRL
1.	Contact [forschungsdaten@zbmed.de](forschungsdaten@zbmed.de)
2.	You will receive:
    * The Terms of use that you need to sign and send back
    * A record sheet that you need to fill in with your metadata and send back
3.	Your metadata will be checked, entered into the FRL and assigned a DOI
4.	Send your data and supplementary materials to the FRL team 
5.	Your data will be added to the metadata record already created in the FRL
6.	You will receive a publication notice with the DOI 

#### Benefits and drawbacks
* **Benefits**
    * Open Access
    * Support of the [FAIR data principles](https://www.go-fair.org/fair-principles/) (e.g. DOI assignment, life science specific metadata)
    * Indexed in [re3data](https://www.re3data.org/)
    * No charges to publish, archive or use the scientific texts and research data
    * Permanent accessibility online
    * You can publish your metadata and receive a DOI in advance of publishing your data
    * Embargo of up to 24 months for specific cases
    * Findable via Google search and indexed in [BASE](https://www.base-search.net/about/en/index.php), [DataCite Search](https://search.datacite.org/) and  [LIVIVO - ZB MED Search Portal for Life Sciences](https://www.livivo.de/)
    * Possibility of publishing big datasets
* **Drawbacks**
    * Not suitable for sensitive data
    * Can only be browsed in German

## Data publishing in Zenodo

#### Repository details
* **Data type**: all research outputs, positive and negative results
* **Open Access**: yes
* **Terms of use & License**: yes
* **Persistent Identifier (PID) system**: yes (DOI)
* **Certification or repository standard**: no
* **Policy**: yes (see [here](https://about.zenodo.org/policies/))
* **Archiving**: yes
* **Costs**: free of charge
* **Data curation & quality control**: by the repository itself or by communities
* **Data guarantee**: availability for the lifetime of the repository
* **Recommended publication formats for research data**: any file format

#### Steps to deposit data in Zenodo 
1.	Upload
2.	Describe
3.	Publish

For more details, see this [guide](https://www.openaire.eu/zenodo-guide)).

#### Benefits and drawbacks
* **Benefits**
  * Open Access
  * Support of the [FAIR data principles](https://www.go-fair.org/fair-principles/) (e.g. DOI assignment)
  * Indexed in [re3data](https://www.re3data.org/)
  * No charges to publish, archive or use the scientific texts and research data
  * Permanent accessibility online
  * Files may be deposited under closed, open, or embargoed access
  * Findable via Google search and indexed in [BASE](https://www.base-search.net/about/en/index.php) and [DataCite Search](https://search.datacite.org/)
  * Size limit of 50 GB per record. For bigger size, a request has to be made.
* **Drawbacks**
    * Not suitable for personal-related / sensitive data that might have to stay in their country of origin.

## Repository finders
* To find any repositories:
    * The DFG-funded registry of research data repositories ([re3data.org](https://www.re3data.org/), [search results](https://www.re3data.org/search?query=Microbiology) for "Microbiology")
    * The RDA-endorsed [FAIRsharing.org](https://fairsharing.org/) ([search results](https://fairsharing.org/search?fairsharingRegistry=Database&subjects=microbiology&page=1) for Databases in Microbiology)
* To find Open Access repositories in the life sciences: [ELIXIR Deposition Databases for Biomolecular Data](https://elixir-europe.org/platforms/data/elixir-deposition-databases)
* To find a suitable repository in the life sciences: [PUBLISSO – Repository Finder](https://www.publisso.de/en/research-data-management/publishing/publisso-repository-finder/)
* To find a suitable interdisciplinary repository: [Generalist Repository Comparison Chart](https://doi.org/10.5281/zenodo.3946720)
* To find Open Access repositories: [OpenDOAR](https://v2.sherpa.ac.uk/opendoar/): Directory of Open Access Repositories

## References
* Engelhardt, Claudia, Biernacka, Katarzyna, Coffey, Aoife, Cornet, Ronald, Danciu, Alina, Demchenko, Yuri, Downes, Stephen, Erdmann, Christopher, Garbuglia, Federica, Germer, Kerstin, Helbig, Kerstin, Hellström, Margareta, Hettne, Kristina, Hibbert, Dawn, Jetten, Mijke, Karimova, Yulia, Kryger Hansen, Karsten, Kuusniemi, Mari Elisa, Letizia, Viviana, … Zhou, Biru. (2022). D7.4 How to be FAIR with your data. A teaching and training handbook for higher education institutions (V1.2.1) [Computer software]. Zenodo. [https://doi.org/10.5281/zenodo.6674301](https://doi.org/10.5281/zenodo.6674301)
* Lindlar, Michelle, Rudnik, Pia, Horton, Laurence, & Jones, Sarah. (2020). "You say potato, I say potato" - Mapping Digital Preservation and Research Data Management Concepts towards Collective Curation and Preservation Strategies. International Digital Curation Conference (IDCC), Dublin. [https://doi.org/10.5281/zenodo.3672773](https://doi.org/10.5281/zenodo.3672773)
* Rathmann T. et al. 2021-10. Workshop on Research Data Management. FoDaKo and ZB MED - Information Centre for Life Sciences. Google Slides.
