---
title: Data Repositories
category: Research-Data-Management
layout: docs_home
docs_css: markdown
---
# Selection of data to be published
To select data to be published, consider the following criteria: 
* Funder requirements
* Your data underly a publication
* You think your data have a high reuse potential: quality, historical importance, uniqueness (i.e. the data contain non-repeatable observations), originality, size, scale, costs of data production or innovative nature of the research
* Existence of functional reusability: can the data be read and reused? Are metadata available and are they sufficient to enable future users to understand your data? Are there any legal objections which prevent the data from being published?

# Selection of a trusted repository

## Criteria
* Does the repository provide:
    * **Open Access** to non-sensitive data?
    * The **terms of use** and **licenses** of the data?
    * A **policy** to help researchers determine whether their data needs will be met?
* Does the repository use a persistent identifier system such as assigning a **Digital Object Identifier** (DOI) to submitted datasets?
* Is the repository **certified** or support a repository **standard**?
* Further criteria: costs, metadata to be provided, repository size, data upload restriction(s), landing pages, guidance on data citation, long-term sustainability

A potential repository should optimally widely used (in similar publications in the field) while offering a scope that is as specific as possible (supporting most specific metadata schemas).

## In order of preference
1. A well-established repository in your discipline 
2. Your institution’s repository (if available)
3. One of the repository recommended by your funding organization
4. A cost-free interdisciplinary repository (e.g. [Figshare](https://figshare.com/), [Zenodo](https://zenodo.org/) (open source))
5. Another repository that you can search for using the above-mentioned criteria in a repository finder. 

## Repository finders
* To find any repositories:
    * The DFG-funded registry of research data repositories ([re3data.org](https://www.re3data.org/), [search results](https://www.re3data.org/search?query=Microbiology) for "Microbiology")
    * The RDA-endorsed [FAIRsharing.org](https://fairsharing.org/) ([search results](https://fairsharing.org/search?fairsharingRegistry=Database&subjects=microbiology&page=1) for Databases in Microbiology)
* To find Open Access repositories in the life sciences: [ELIXIR Deposition Databases for Biomolecular Data](https://elixir-europe.org/platforms/data/elixir-deposition-databases)
* To find a suitable repository in the life sciences: [PUBLISSO – Repository Finder](https://www.publisso.de/en/research-data-management/publishing/publisso-repository-finder/)
* To find a suitable interdisciplinary repository: [Generalist Repository Comparison Chart](https://doi.org/10.5281/zenodo.3946720)
* To find Open Access repositories: [OpenDOAR](https://v2.sherpa.ac.uk/opendoar/): Directory of Open Access Repositories

## Well-established repositories in microbiology
Primary raw data from experiments studying the microbiota consists in most cases of either sequence data, mass spectra or images and derived results.

* **Crystallographic data**: Cambridge Structural Database ([CSD](https://www.ccdc.cam.ac.uk/solutions/csd-core/components/csd/))
* **Geospatial data**: Malaria Atlas Project ([MAP](https://malariaatlas.org/))
* **Image data**: 
  * [BioImage Archive](https://www.ebi.ac.uk/bioimage-archive/) for submission of most imaging techniques (especially microscopy) from molecules to organisms.
  * [Cell Image Library](http://www.cellimagelibrary.org/home)
* **Linked genotype and phenotype data**: European Genome-phenome Archive ([EGA](https://ega-archive.org/))
* **Linked Open Data (LOD)**: [The Linked Open Data Cloud](https://lod-cloud.net/)
* **Macromolecular structures**: 
  * Worldwide Protein Data Bank ([wwPDB](http://www.wwpdb.org/))
  * Biological Magnetic Resonance Data Bank ([BMRB](https://bmrb.io/))
* **Electron microscopy data**:
  * Electron Microscopy Data Bank ([EMDB](https://www.ebi.ac.uk/emdb/))
  * Electron Microscopy Public Image Archive ([EMPIAR](https://www.ebi.ac.uk/empiar/)) for most electron microscopy (EM) derived data.
* **Microbiome data**: [MGnify](https://www.ebi.ac.uk/metagenomics/)
* **Nucleic acid sequences** (raw sequencing data (reads or traces), genome assemblies, annotated sequences, sample metadata):
  * [GenBank](https://www.ncbi.nlm.nih.gov/genbank/)
  * DNA Data Bank of Japan ([DDBJ](https://www.ddbj.nig.ac.jp/index-e.html))
  * European Nucleotide Archive ([ENA](https://www.ebi.ac.uk/ena/browser/home)) which hosts world-wide data from raw sequencing runs to sequence assembly information and functional annotation.
  * Sequence Read Archive ([SRA](https://www.ncbi.nlm.nih.gov/sra)) where submissions are structured by BioProject -> SRA Study -> BioSample -> Run.
  * Genome Sequence Archive ([GSA](https://ngdc.cncb.ac.cn/gsa/))
  * **Metagenomes**: [MG-RAST](https://www.mg-rast.org/)
  * **Genetic polymorphism**: European Variation Archive ([EVA](https://www.ebi.ac.uk/eva/))
  * **Functional genomics** / **gene expression data**:
    * Gene Expression Omnibus ([GEO](https://www.ncbi.nlm.nih.gov/geo/))
    * [ArrayExpress](https://www.ebi.ac.uk/biostudies/arrayexpress)
    * [GenomeRNAi](http://www.genomernai.org/)
    * European Genome-phenome Archive ([EGA](https://ega-archive.org/))
    * Database of Interacting Proteins ([DIP](https://dip.doe-mbi.ucla.edu/dip/Main.cgi))
    * [IntAct](https://www.ebi.ac.uk/intact/home)
    * Japanese Genotype-phenotype Archive ([JGA](https://www.ddbj.nig.ac.jp/jga/index-e.html))
    * [PubChem](https://pubchem.ncbi.nlm.nih.gov/)
    * Genomic Expression Archive ([GEA](https://www.ddbj.nig.ac.jp/gea/index-e.html))
    * Genome-Wide Association Studies ([GWAS](https://www.ebi.ac.uk/gwas/)) Catalog
* **Protein sequences**: [UniProt](https://www.uniprot.org/)
  * **Protein-protein interactions**: [STRING](https://string-db.org/)
  * **Proteomes**: PRoteomics IDEntifications ([PRIDE](https://wwwdev.ebi.ac.uk/pride/)) Archive database for submission of protein and peptide data from raw mass spectra and subsequent analysis steps.
* **Quantitative and predictive food microbiology**: [ComBase](https://www.combase.cc/index.php/en/)
* **Scientific texts**: [PUBLISSO – Repository for Life Sciences](https://repository.publisso.de/)
* **Species interaction data**: Global Biotic Interactions ([GloBI](https://www.globalbioticinteractions.org/))
* **Standardized bacterial information**: Bacterial Diversity Metadatabase ([BacDive](https://bacdive.dsmz.de/))
* **Vertebrate-virus network**: [VIRION](https://viralemergence.github.io/virion/)

# Benefits of publishing your research data in a data repository
* Back-up 
* Increased data discoverability and accessibility, and thus potential data reuse
* Increased visibility and citations as repositories can be harvested by search engines such as:
    * [BASE](https://www.base-search.net/about/en/index.php)
    * [DataCite Search](https://search.datacite.org/)
    * [Google Dataset Search](https://datasetsearch.research.google.com/)
    * [LIVIVO - ZB MED Search Portal for Life Sciences](https://www.livivo.de/)
