---
title: Data Repositories
category: Research-Data-Management
layout: docs_home
docs_css: markdown
---
# Selection and preparation of data to be published

# Selection of a trusted repository

## Criteria
A potential repository should optimally widely used (in similar publications in the field) while offering a scope that is as specific as possible (supporting most specific metadata schemas).
Researchers should check whether the repository supports **open access** for non-sensitive data and provides persistent identifiers for submitted datasets.

## In order of preference

## Repository finders
Databases and data repositories in microbiology can be found in the two following registries:
* [re3data.org](https://www.re3data.org/) ([search results](https://www.re3data.org/search?query=Microbiology) for "Microbiology")
* [FAIRsharing.org](https://fairsharing.org/) ([search results](https://fairsharing.org/search?fairsharingRegistry=Database&subjects=microbiology&page=1) for Databases in Microbiology)

## Well-established repositories in microbiology
Primary raw data from experiments studying the microbiota consists in most cases of either sequence data, mass spectra or images and derived results.

* **Crystallographic data**: Cambridge Structural Database ([CSD](https://www.ccdc.cam.ac.uk/solutions/csd-core/components/csd/))
* **Geospatial data**: Malaria Atlas Project ([MAP](https://malariaatlas.org/))
* **Image data**: 
  * [BioImage Archive](https://www.ebi.ac.uk/bioimage-archive/) for submission of most imaging techniques (especially microscopy) from molecules to organisms.
  * [Cell Image Library](http://www.cellimagelibrary.org/home)
* **Linked genotype and phenotype data**: European Genome-phenome Archive ([EGA](https://ega-archive.org/))
* **Linked Open Data (LOD)**: [The Linked Open Data Cloud](https://lod-cloud.net/)
* **Macromolecular structures**: 
  * Worldwide Protein Data Bank ([wwPDB](http://www.wwpdb.org/))
  * Biological Magnetic Resonance Data Bank ([BMRB](https://bmrb.io/))
* **Electron microscopy data**:
  * Electron Microscopy Data Bank ([EMDB](https://www.ebi.ac.uk/emdb/))
  * Electron Microscopy Public Image Archive ([EMPIAR](https://www.ebi.ac.uk/empiar/)) for most electron microscopy (EM) derived data.
* **Microbiome data**: [MGnify](https://www.ebi.ac.uk/metagenomics/)
* **Nucleic acid sequences** (raw seqeuncing data (reads or traces), genome assemblies, annotated sequences, sample metadata):
  * [GenBank](https://www.ncbi.nlm.nih.gov/genbank/)
  * DNA Data Bank of Japan ([DDBJ](https://www.ddbj.nig.ac.jp/index-e.html))
  * European Nucleotide Archive ([ENA](https://www.ebi.ac.uk/ena/browser/home)) which hosts world-wide data from raw sequencing runs to sequence assembly information and functional annotation.
  * Sequence Read Archive ([SRA](https://www.ncbi.nlm.nih.gov/sra)) where submissions are structured by BioProject -> SRA Study -> BioSample -> Run.
  * Genome Sequence Archive ([GSA](https://ngdc.cncb.ac.cn/gsa/))
  * **Metagenomes**: [MG-RAST](https://www.mg-rast.org/)
  * **Genetic polymorphism**: European Variation Archive ([EVA](https://www.ebi.ac.uk/eva/))
  * **Functional genomics** / **gene expression data**:
    * Gene Expression Omnibus ([GEO](https://www.ncbi.nlm.nih.gov/geo/))
    * [ArrayExpress](https://www.ebi.ac.uk/biostudies/arrayexpress)
    * [GenomeRNAi](http://www.genomernai.org/)
    * European Genome-phenome Archive ([EGA](https://ega-archive.org/))
    * Database of Interacting Proteins ([DIP](https://dip.doe-mbi.ucla.edu/dip/Main.cgi))
    * [IntAct](https://www.ebi.ac.uk/intact/home)
    * Japanese Genotype-phenotype Archive ([JGA](https://www.ddbj.nig.ac.jp/jga/index-e.html))
    * [PubChem](https://pubchem.ncbi.nlm.nih.gov/)
    * Genomic Expression Archive ([GEA](https://www.ddbj.nig.ac.jp/gea/index-e.html))
    * Genome-Wide Association Studies ([GWAS](https://www.ebi.ac.uk/gwas/)) Catalog
* **Protein sequences**: [UniProt](https://www.uniprot.org/)
  * **Protein-protein interactions**: [STRING](https://string-db.org/)
  * **Proteomes**: PRoteomics IDEntifications ([PRIDE](https://wwwdev.ebi.ac.uk/pride/)) Archive database for submission of protein and peptide data from raw mass spectra and subsequent analysis steps.
* **Quantitative and predictive food microbiology**: [ComBase](https://www.combase.cc/index.php/en/)
* **Scientific texts**: [PUBLISSO â€“ Repository for Life Sciences](https://repository.publisso.de/)
* **Species interaction data**: Global Biotic Interactions ([GloBI](https://www.globalbioticinteractions.org/))
* **Standardized bacterial information**: Bacterial Diversity Metadatabase ([BacDive](https://bacdive.dsmz.de/))
* **Vertebrate-virus network**: [VIRION](https://viralemergence.github.io/virion/)

# Preprint servers

# Benefits of publishing your research data

# Further resources

# References
